lib/jnpr/jsnapy/jsnapy.py

```diff
@@ -1049,9 +1063,9 @@ def check_arguments(self):
             self.parser.print_help()
             sys.exit(1)
 
-        if((self.args.snap is True and (self.args.pre_snapfile is None or self.args.file is None)) or
+        if(((self.args.snap is True and (self.args.pre_snapfile is None or self.args.file is None)) or
             (self.args.snapcheck is True and self.args.file is None) or
-            (self.args.check is True and self.args.file is None)
+            (self.args.check is True and self.args.file is None)) and (self.args.test_file is None or self.args.hostname is None or self.args.login is None or self.args.passwd is None)
```

Comment:
You code violates pep8 (line not more than 80 char, can we indent code properly.